.solhint.json

```diff
@@ -15,6 +15,7 @@
     "constructor-syntax": "error",
     "func-visibility": ["error", {"ignoreConstructors": true}],
     "not-rely-on-time": "off",
+    "no-inline-assembly": "off",
```

Comment:
that was pushed by mistake....fixed -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/417/commits/a7ee4a2f81e1acb2f1036e1d55f586eb54f26472

Reply:
This is needed because of the following assembly code:
https://github.com/thesandboxgame/sandbox-smart-contracts/blob/a7ee4a2f81e1acb2f1036e1d55f586eb54f26472/src/solc_0.8/common/Libraries/SigUtil.sol#L10

https://github.com/thesandboxgame/sandbox-smart-contracts/blob/a7ee4a2f81e1acb2f1036e1d55f586eb54f26472/src/solc_0.8/common/Libraries/SigUtil.sol#L34